RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -70,7 +70,7 @@ struct PurchaseInformation {
 
     /// Indicates whether the purchased subscription is active
     ///
-    /// Note: `false` for non-subscriptions
```

Comment:
cc @vegaro a non-sub should always be active, right?

Reply:
Ok then let's do as discussed, and rename this to `isExpired` instead. I'll apply it in another PR